contracts/RelayerOracleConsumer.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "./interface/CLV2V3Interface.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract RelayerOracleConsumer is Ownable {
+    CLV2V3Interface public priceFeed;
+    CLV2V3Interface public relayerFeed;
+    uint256 public deviationPercent = 50; // 0.5%
+    uint256 public maxDelay = 15 minutes;
```

Comment:
Shouldn't those parameters be part of the constructor?
The deviation and maximum delay may vary depending on the price feed itself and the use case at hand.